ticketauth/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("UnstableApiUsage")
```

Comment:
Why is this needed?

Reply:
apparently if you use agp 7.x.x with latest version of Android Studio a lot of the regular functions like minSdk etc are marked @incubating, which they are gonna be for half a year or something, so I suppressed them for now. Let me know if you know a better solution or if I got @incubating wrong :)